daemons/controld/controld_execd.c

```diff
@@ -2666,6 +2666,7 @@ log_executor_event(const lrmd_event_data_t *op, const char *op_key,
         case PCMK_EXEC_CANCELLED:
             log_level = LOG_INFO;
             // Fall through
+	    __attribute__((fallthrough));
```

Comment:
Huh, I thought we found out there is no macro or compiler directive that works for both GCC and Clang. See also https://projects.clusterlabs.org/T558

I hope this works :)

Reply:
it does actually work with recent gcc, but it fails with gcc on rhel7. There are other workarounds for that.